add verbiage about use_device_id_as_hostname since it is required for network device tagging to work (#33600)
DataDog/documentation: content/en/integrations/guide/servicenow-cmdb-enrichment-setup.md

1. Select any optional field name remappings.
1. Click **Save**.

> **Note**: In order for tags ingested through the CMDB query, your `conf.yaml` used for polling your network devices must have the parameter `use_device_id_as_hostname` set to `true`.

Reviewer: Since it sounds like this configuration should be in place **before** a user completes the steps listed above, do you think this note might be better situated above those steps, perhaps where line `68` is currently? 

Additionally, we can potentially make this note more eye-catching and easy to notice by adding it to an info div. I've tried this in a local build of the documentation site, screenshot below:
<img width="766" height="280" alt="device_tagging_note" src="https://github.com/user-attachments/assets/5329399f-b4df-4cea-864f-30a8b8e76633" />

If you think this might be a more effective style, this can be added with the following snippet:
`<div class="alert alert-info">You must set <code>use_device_id_as_hostname: true</code> in the Datadog Agent's <code>snmp.d/conf.yaml</code> file to ingest network device tags.</div>`

- The user who created and executes the Query Builder query matches a username in your Datadog configuration. The user in ServiceNow must have the role `cmdb_query_builder_read`.
- The number of results returned by your query must be less than or equal to the `glide.cmdb.query.max_results_limit` setting in ServiceNow. By default, the maximum number of results is 10000. To change the setting, go to **Configuration** > **CMDB Properties** > **Query Builder Properties**.
- All CIs configured in your Query Builder query must have a `1` label. This ensures you have not created any duplicate CIs, which the parser does not support.

> - The configuration for the agent polling the network devices has `use_device_id_as_hostname` set to `true`.

Reviewer: ```suggestion
- The Agent polling the network devices is configured with `use_device_id_as_hostname: true`.
```
